integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -477,11 +477,12 @@ class CarbonFileMetastore extends CarbonMetaStore {
       // clear driver B-tree and dictionary cache
       ManageDictionaryAndBTree.clearBTreeAndDictionaryLRUCache(carbonTable)
     }
+    // Trying to clear the cache first before dropping the table
```

Comment:
do not say trying to clear, just write clear both driver and executor cache

Reply:
done